src/accountsdb/cache.zig

```diff
@@ -13,6 +13,8 @@ const Slot = sig.core.Slot;
 pub const AccountsCache = struct {
     lru: LRU,
     allocator: std.mem.Allocator,
+    cache_hits: usize = 0,
+    cache_misses: usize = 0,
```

Comment:
would probs be best to make these use prometheus counters

Reply:
https://github.com/Syndica/sig/pull/325/commits/85ec8c3441692383fed14ff3f66f0c71e1cd35df#diff-7f62328c8d0fd51e206b91ff2b72aef34de0fae176ba46106a0c768d29f306f8R14-R18